packages/web-components/src/select/select.styles.ts

```diff
@@ -111,6 +128,7 @@ export const selectStyles = (context, definition) =>
       width: 100%;
       z-index: 1;
       margin: 1px 0;
+      border: 1px solid transparent;
```

Comment:
Should this be strokeWidth and does the padding need to be reduced to maintain the same size?

Reply:
updated